src/usdb_syncer/json.py

```diff
@@ -0,0 +1,79 @@
+"""Generates a JSON from the passed song list."""
+
+import datetime
+import json
+from typing import Any, Iterable
+
+from usdb_syncer import SongId
+from usdb_syncer.constants import VERSION
+from usdb_syncer.logger import get_logger
+from usdb_syncer.resource_dl import _url_from_resource
+from usdb_syncer.song_txt.headers import Headers
+from usdb_syncer.usdb_scraper import UsdbSong
+
+_logger = get_logger(__file__)
+
+MAX_SIZE = (100, 100)
+
+
+def get_headers(txt_path: str) -> Headers:
+    with open(txt_path, "r", encoding="utf-8") as file:
+        lines = [line for line in file.read().splitlines() if line]
+        return Headers.parse(lines, _logger)
```

Comment:
`year`, `edition`, `genre` were not available internally in the syncer in my first atempt end of December - is it now? Thus I reparsed the txt and as I already did so also used the `artist` and `title` from the txt as well to get any user specific changes.

Reply:
All attributes from the USDB song list have always been available. This includes edition, but not year and genre. It would make sense to store these attributes somewhere after downloading a new txt.
But if we now start to dynamically read them from disk in some parts of the code, it will be harder to unify later on.
Most songs don't have these attributes anyway. Can't we just leave them out for now?

Reply:
IMHO they appear relevant for a song list export. Otherwise the ability to sort/filter songs for `genre` or `year` relies on an additional fetch from usdb.
I could try to extend `UsdbSong` accordingly so that it contains `genre` and `year`. We might also need to add `tags` once they are supported in one way or the other (not now)
There is already an if condition in place that handles possible inavailability of the values.

Reply:
I can add support for these attributes when I get a chance. You could set them to `None` for now, and we populate them as soon as the syncer supports it.

Reply:
done in 1a2db45